### PR TITLE
Add entry point firebase/messaging/sw

### PR DESCRIPTION
**Pull request by Feiyang1**
Changes:

- Add entry point firebase/messaging/sw
- Remove CDN builds for modular packages (we only provide CDN builds for compat packages)

### CODE REVIEW

**Review by hsubox76 — APPROVED** [review]

File: packages-exp/messaging-exp/src/api/isSupported.ts [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
 import { validateIndexedDBOpenable } from '@firebase/util';
 
+/**
+ * Check if all required APIs exist in the browser.
```
Comment (zwu52) [inline packages-exp/messaging-exp/src/api/isSupported.ts:21]: Checks

File: packages-exp/messaging-exp/src/api/isSupported.ts [hunk 2]
```diff
@@ -36,7 +42,10 @@ export async function isWindowSupported(): Promise<boolean> {
 }
 
 /**
- * Checks to see if the required APIs exist within SW Context.
+ * Check whether all required APIs exist within SW Context
```
Comment (zwu52) [inline packages-exp/messaging-exp/src/api/isSupported.ts:45]: ditto

File: packages-exp/messaging-exp/src/api.ts [hunk 3]
```diff
@@ -68,29 +72,15 @@ export function getMessagingInSw(
...
- * See
- * {@link https://developers.google.com/web/fundamentals/push-notifications/web-push-protocol | The Web Push Protocol}
- * for details on web push services.
- *
- * @param options.serviceWorkerRegistration - The service worker registration for receiving push
- * messaging. If the registration is not provided explicitly, you need to have a
- * `firebase-messaging-sw.js` at your root location. See
- * {@link https://firebase.google.com/docs/cloud-messaging/js/client#retrieve-the-current-registration-token | Retrieve the current registration token}
- * for more details.
+ * @param options - Provides an optional vapid key and an optinoal service worker registration
```
Comment (zwu52) [inline packages-exp/messaging-exp/src/api.ts:75]: nit: "VAPID"

**Review by zwu52 — CHANGES_REQUESTED** [review]
Cool! Thanks for the changes here 👍  some nits

**Review by zwu52 — APPROVED** [review]

**Pull request merged**